spec/integration/inventory_spec.rb

```diff
@@ -110,36 +131,36 @@ module Hawkular::Inventory::RSpec
     end
 
     it 'Should list children of WildFly' do
-      resources = @client.list_resources_for_type(@state[:feed], 'WildFly Server')
+      resources = @client.list_resources_for_type(feed_id, 'WildFly Server')
       expect(resources.size).to be(1)
 
       wild_fly = resources[0]
 
       children = @client.list_child_resources(wild_fly)
 
-      expect(children.size).to be(21)
+      expect(children.size).to be(22)
     end
 
     it 'Should list recursive children of WildFly' do
-      resources = @client.list_resources_for_type(@state[:feed], 'WildFly Server')
+      resources = @client.list_resources_for_type(feed_id, 'WildFly Server')
       wild_fly = resources[0]
 
       children = @client.list_child_resources(wild_fly, recursive: true)
 
-      expect(children.size).to be(211)
+      expect(children.size).to be(251)
```

Comment:
So do you suggest doing something like 

```ruby
expect(children.size).to be > x
```
instead?

As it is, the assumptions are stronger, it's rather question of taste what is better, I can argue that MiQ has the same thing for testing the # of features, that needs to be changed almost everytime and on other places there is something vague like `expect(something.size).to be >= 1 `

But to be honest, I don't really care, I can change that to `> x`